integration-test/index.ts

```diff
@@ -17,27 +12,27 @@ const palette = {
   Primary: "var(--primary)",
 };
 
-const fonts = {
+const fonts: FontConfig = {
   f24: "24px",
   f18: "18px",
   f16: "16px",
   f14: "14px",
   f12: "12px",
-  f10: { "fontSize": "10px", "fontWeight": 500 },
+  f10: { fontSize: "10px", fontWeight: 500 },
 };
 
 const breakpoints = { sm: 0, md: 600, lg: 960 };
 
-const methods = generateRules({ palette, fonts, numberOfIncrements });
```

Comment:
Got ride of the "methods" name and using "sections" instead.

And now "sections" is a map of "section name -> RuleFn" just like the regular `defaultRuleFns` are inside of truss.